samples/snippets/src/main/java/com/example/mediatranslation/TranslateFromFile.java

```diff
@@ -0,0 +1,93 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.mediatranslation;
+
+// [START media_translation_translate_from_file]
+
+import com.google.api.gax.rpc.BidiStream;
+import com.google.cloud.mediatranslation.v1beta1.SpeechTranslationServiceClient;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechConfig;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechRequest;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResponse;
+import com.google.cloud.mediatranslation.v1beta1.StreamingTranslateSpeechResult;
+import com.google.cloud.mediatranslation.v1beta1.TranslateSpeechConfig;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class TranslateFromFile {
+
+  public static void translateFromFile() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String filePath = "path/to/audio.raw";
+    translateFromFile(filePath);
+  }
+
+  public static void translateFromFile(String filePath) throws IOException {
+    try (SpeechTranslationServiceClient speechTranslationServiceClient =
+        SpeechTranslationServiceClient.create()) {
+      Path path = Paths.get(filePath);
+      byte[] content = Files.readAllBytes(path);
+      ByteString audioContent = ByteString.copyFrom(content);
+
+      TranslateSpeechConfig translateSpeechConfig =
+          TranslateSpeechConfig.newBuilder()
+              .setAudioEncoding("linear16")
+              .setSampleRateHertz(16000)
+              .setSourceLanguageCode("en-US")
+              .setTargetLanguageCode("fr-FR")
+              .build();
+
+      StreamingTranslateSpeechConfig config =
+          StreamingTranslateSpeechConfig.newBuilder().setAudioConfig(translateSpeechConfig).build();
+
+      BidiStream<StreamingTranslateSpeechRequest, StreamingTranslateSpeechResponse> bidiStream =
+          speechTranslationServiceClient.streamingTranslateSpeechCallable().call();
+
+      // The first request contains the configuration.
+      StreamingTranslateSpeechRequest requestConfig =
+              StreamingTranslateSpeechRequest.newBuilder().setStreamingConfig(config).build();
+
+      // The second request contains the audio
+      StreamingTranslateSpeechRequest request =
+              StreamingTranslateSpeechRequest.newBuilder().setAudioContent(audioContent).build();
+
+      bidiStream.send(requestConfig);
+      bidiStream.send(request);
+
+      for (StreamingTranslateSpeechResponse response : bidiStream) {
+        // Once the transcription settles, the response contains the
+        // is_final result. The other results will be for subsequent portions of
+        // the audio.
```

Comment:
Does the config have to set anything to enable partial transcription?

Reply:
I asked prod team eng - he said: they dont support partial transcription

Reply:
They don't support partial transcription? 
But below your print the partial translation?

Reply:
I think they support partial transcription, I do not know why he said that. He may have misunderstood my question.

Partial transcription actually works in this sample.